src/reactions/halloween.ts

```diff
@@ -0,0 +1,34 @@
+import { logger } from '../utilities/logger';
+import { ReactionDef } from './reaction-def';
+
+export const halloweenReaction: ReactionDef = {
+  emoji: '🎃',
+  description: 'Adds the halloween role to the user who reacts',
```

Comment:
We should update this with more information for the command and event, like what does the Halloween role do?

Reply:
Added additional description.